test/ooni/http_invalid_request_line_test.cpp

```diff
@@ -76,3 +76,16 @@ TEST_CASE("Make sure that it can pass options to the other levels") {
     } while (!done);
     for (auto &s : *logs) std::cout << s << "\n";
 }
+
+TEST_CASE("Make sure that the test can deal with an invalid backend") {
+    Var<std::list<std::string>> logs(new std::list<std::string>);
+    bool done = false;
+    ooni::HttpInvalidRequestLineTest()
+        .set_backend("nexacenter.org")
+        .on_log([=](const char *s) { logs->push_back(s); })
+        .run([&done]() { done = true; });
```

Comment:
Yes, I used the implementation of the test used [here](https://github.com/measurement-kit/measurement-kit/blob/fix/http_invalid_request_line_backend/test/ooni/http_invalid_request_line_test.cpp#L36). The blocking and non blocking version are actually the same. I'm going to use the blocking version to more concise!